noggin/tests/unit/controller/test_user_otp.py

```diff
@@ -525,23 +525,23 @@ def test_user_settings_otp_enable(client, logged_in_dummy_user, dummy_user_with_
     # grab the id of the first token
     tokenid = tokenlist[0].select(".text-monospace")[0].get_text(strip=True)
 
-    # disable that token
+    # delete that token
     result = client.post(
-        "/user/dummy/settings/otp/disable/",
+        "/user/dummy/settings/otp/delete/",
```

Comment:
I don't understand this change and the rest in the test, it looks like you're testing something else.

Reply:
The new `ClientMeta` seems to have changed the semantics of `otptoken_mod()` slightly, i.e. setting `o_ipatokendisabled=None` won't reset it to the default (`False`) anymore, which broke enabling disabled OTP tokens, thus the modifications to the test (deleting a token worked :wink:). I've fixed how the controller method invokes `otptoken_mod()` and reverted the changes to the test in the latest push.